internal/dataplane/kong_client_test.go

```diff
@@ -246,9 +251,10 @@ func (m *mockUpdateStrategy) Type() string {
 	return "Mock"
 }
 
-// mockConfigurationChangeDetector is a mock implementation of sendconfig.ConfigurationChangeDetector.
+// mockConfigurationChangeDetector is a mock implementation of sendconfig.ConfigurationGetter.
 type mockConfigurationChangeDetector struct {
 	hasConfigurationChanged bool
+	status                  kong.Status
```

Comment:
As far as I can see it's not used anywhere. I think this mock should just have the `hasConfigurationChanged` property which tells it what to return from `HasConfigurationChanged`.

Reply:
yep, removed